crates/celestia/celestia-test-components/src/bootstrap/impls/generate_wallet_config.rs

```diff
@@ -80,12 +80,22 @@ where
             validator_staked_amount: None,
         };
 
+        let relayer_2 = CosmosWalletConfig {
+            wallet_id: "relayer-2".to_owned(),
+            genesis_balances: vec![
+                Amount::new(1_000_000_000_000_000_000, denom_for_staking.clone()),
+                Amount::new(1_000_000_000_000_000_000, denom_for_transfer.clone()),
+            ],
+            validator_staked_amount: None,
+        };
+
         Ok(BTreeMap::from([
             ("validator".into(), validator),
             ("sequencer".into(), sequencer),
             ("user1".into(), user1),
             ("user2".into(), user2),
             ("relayer".into(), relayer),
```

Comment:
can we have `relayer1` and `relayer2` as `user1` and `user2`?

Reply:
sorry, I meant just rename `relayer` to `relayer1` to have convention of `relayerN`, same as `userN`.

Reply:
Done in 5a4c39ca1b4184be2580769b664b076aae7b6f25